### PR TITLE
docs: Add How-to guides section with the first guide

### LINKED ISSUE #7854: docs: Add How-to-guides section to APISIX Website API Gateway
### Current State

Currently, we have categorization based on plugins but there is no separation by usability cases.

Actually, plugins cover many things but It is hard to navigate through an exact fit plugin for some use cases without opening the individual plugin tutorial.

For newbies will be quite challenging to understand the core use cases of API Gateway.

### Desired State

We should have a How-to-guides section with learning content per each API Gateway use case that includes the following subpages:

Expose APIs
Protect APIs
Secure API
Monitor APIs
Improve performance. 
Health Check.
Add multiple versions. 
Cache API responses. 
Mock API responses.
Canary release. 
.... more

We have already enough blog posts that cover many API Gateway use cases we can rewrite them for each how-to guide page.

Here is the link to issue #6461

https://github.com/apache/apisix/issues/6461

You can see examples for Azure API Management or Kong API Gateway how the content is structured:

image

image

### PR DESCRIPTION
**Pull request by Boburmirzo**
### Description

Added How-To guide section with the first guide for Observe your APIs which shows use cases of observability plugins. 

Fixes #7854 

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+title: Observer your APIs
+keywords:
+  - API gateway
+  - Apache APISIX
+  - Observability
+  - Monitor
+  - Plugins
+description: Apache APISIX Observability Plugins and take a look at how to set up these plugins.
+---
+
```
Comment (hf400159) [inline docs/en/latest/how-to-guides/observe-your-api.md:11]: please add ASF 2.0 license. please refer to: https://github.com/apache/apisix/edit/master/docs/en/latest/plugins/api-breaker.md

Comment (hf400159) [reply docs/en/latest/how-to-guides/observe-your-api.md:11]: image

**Review by hf400159 — COMMENTED** [review]
great! 👍

File: docs/en/latest/how-to-guides/observe-your-api.md (hunk 1 above)
Comment (Boburmirzo) [reply docs/en/latest/how-to-guides/observe-your-api.md:11]: @hf400159 Right, I added now license description.

**Comment by Boburmirzo** [comment]
@juzhiyuan We need one more reviewer and approver:)

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+- [google-cloud-logging](https://apisix.apache.org/docs/apisix/plugins/google-cloud-logging)
+
+And you can see the [full list](https://apisix.apache.org/docs/apisix/plugins/zipkin) on the official website of Apache APISIX. Now for demo purposes, let's choose a simple but mostly used _http-logger_ plugin that is capable of sending API Log data requests to HTTP/HTTPS servers or sends as JSON objects to Monitoring tools. We can assume that a route and an upstream are created.  You can learn how to set up them in the **[Getting started with Apache APISIX](https://youtu.be/dUOjJkb61so)** video tutorial. Also, you can find all command-line examples on the GitHub page [apisix-observability-plugins](https://boburmirzo.github.io/apisix-observability-plugins/)
+
+You can generate a mock HTTP server at [mockbin.com](https://mockbin.org/) to record and view the logs. Note that we also bind the route to an upstream (You can refer to this documentation to learn about more [core concepts of Apache APISIX](https://apisix.apache.org/docs/apisix/architecture-design/apisix)).
+
+The following is an example of how to enable the http-logger for a specific route.
+
+```bash
```
Comment (SylviaBABY) [inline docs/en/latest/how-to-guides/observe-your-api.md:89]:
```suggestion
```shell
```

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+
+```json
+{
+  "uri": "http://mockbin.org/bin/5451b7cd-af27-41b8-8df1-282ffea13a61"
+}
+```
+
+Once we get a successful response from APISIX server, we can send a request to this _get_ endpoint to generate logs.
+
+```bash
```
Comment (SylviaBABY) [inline docs/en/latest/how-to-guides/observe-your-api.md:114]:
```suggestion
```shell
```

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+## Metrics
+
+**Metrics** are a numeric representation of data measured over intervals of time. You can also aggregate this data into daily or weekly frequency and run queries against a distributed system like [Elasticsearch](https://www.elastic.co/). Or sometimes based on metrics you trigger alerts to take any action later. Once API metrics are collected, you can track them with metrics tracking tools such as [Prometheus](https://prometheus.io/).
+
+Apache APISIX API Gateway also offers [prometheus-plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) to fetch your API metrics and expose them in Prometheus. Behind the scene, Apache APISIX downloads the Grafana dashboard meta, imports it to [Grafana](https://grafana.com/), and fetches real-time metrics from the Prometheus plugin.
+
+Let’s enable prometheus-plugin for our route:
+
+```bash
```
Comment (SylviaBABY) [inline docs/en/latest/how-to-guides/observe-your-api.md:132]:
```suggestion
```shell
```

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+  "plugins": {
+    "prometheus": {}
+  },
+  "upstream_id": "1"
+}'
+```
+
+We fetch the metric data from the specified URL `/apisix/prometheus/`metrics.
+
+```bash
```
Comment (SylviaBABY) [inline docs/en/latest/how-to-guides/observe-your-api.md:145]:
```suggestion
```shell
```

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+
+## Tracing
+
+The third is **tracing** or distributed tracing allows you to understand the life of a request as it traverses your service network and allows you to answer questions like what service has this request touched and how much latency was introduced. Traces enable you to further explore which logs to look at for a particular session or related set of API calls.
+
+[Zipkin](https://zipkin.io/) an open-source distributed tracing system. [APISIX plugin](https://apisix.apache.org/docs/apisix/plugins/zipkin) is supported to collect tracing and report to Zipkin Collector based on [Zipkin API specification](https://zipkin.io/pages/instrumenting.html).
+
+Here’s an example to enable the _zipkin plugin_ on the specified route:
+
+```bash
```
Comment (SylviaBABY) [inline docs/en/latest/how-to-guides/observe-your-api.md:200]:
```suggestion
```shell
```

File: docs/en/latest/how-to-guides/observe-your-api.md [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+      "sample_ratio": 1
+    }
+  },
+  "upstream_id": "1"
+}'
+```
+
+We can test our example by simply running the following curl command:
+
+```bash
```
Comment (SylviaBABY) [inline docs/en/latest/how-to-guides/observe-your-api.md:219]:
```suggestion
```shell
```

**Comment by Boburmirzo** [comment]
@SylviaBABY Thanks, I updated the requested changes.

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Comment by Boburmirzo** [comment]
@hf400159 One question, do these changes go with the next APISIX release, right?

**Comment by hf400159** [comment]
Yes. You can find it here. https://apisix.apache.org/docs/apisix/next/how-to-guides/observe-your-api/   cc@Boburmirzo

**Comment by Boburmirzo** [comment]
> Yes. You can find it here. https://apisix.apache.org/docs/apisix/next/how-to-guides/observe-your-api/ cc@Boburmirzo image

Great! I will work on yet another guide soon.

**Pull request merged**